androidApp/src/main/java/kmm/rickandmorty/app/android/presentation/components/episodes/EpisodesScreen.kt

```diff
@@ -38,33 +36,42 @@ fun EpisodesScreen() {
     when (state) {
         is Success -> {
             LazyColumn(
-                contentPadding = PaddingValues(top = 10.dp, bottom = 60.dp),
-                verticalArrangement = Arrangement.spacedBy(10.dp)
+                contentPadding = PaddingValues(top = dimensionResource(id = R.dimen.episodes_list_content_top_padding), bottom = dimensionResource(id = R.dimen.episodes_list_content_bottom_padding)),
+                verticalArrangement = Arrangement.spacedBy(
+                    dimensionResource(id = R.dimen.episodes_list_spacer)
+                )
             ) {
                 items((state as Success).data) { episode ->
                     Episode(episode)
                 }
             }
         }
         is Error -> Unit
```

Comment:
Handle error case

Reply:
Will set up error components in a later pull request.